src/net.h

```diff
@@ -677,6 +677,7 @@ class CNode
 
 
     uint64_t nLocalHostNonce;
+    // What local services you advertised on connection to this node
```

Comment:
I think the actors in this sentence are slightly confusing, who is 'you' and who is 'this node' here?